ember-client/shared/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -389,7 +389,7 @@ object EmberClientBuilder extends EmberClientBuilderCompanionPlatform {
     val maxTotal: Int = 100
     val idleTimeInPool: FiniteDuration = 30.seconds // 30 Seconds in Nanos
     val additionalSocketOptions: List[SocketOption] = List.empty[SocketOption]
-    val userAgent: Option[`User-Agent`] = Some(
+    val userAgent: Some[`User-Agent`] = Some(
```

Comment:
Wait, hang on, it looks like this worked? This makes me concerned about what's going on in Scala 3 MiMa which is passing before and after this change ...